torch/testing/_internal/custom_op_db.py

```diff
@@ -352,7 +351,7 @@ def numpy_nms_abstract(boxes, scores, iou_threshold):
 
     ctx = torch._custom_op.impl.get_ctx()
     i0 = ctx.create_unbacked_symint()
-    result = boxes.new_empty([i0, 4])
+    result = boxes.new_empty([i0], dtype=torch.int64)
```

Comment:
Test failures look real. It looks like make_fx(tracing_mode='fake') now doesn't raise DynamicOutputShapeException? 
<img width="905" alt="image" src="https://github.com/pytorch/pytorch/assets/5652049/ad6f001a-f902-495c-956a-b35071e529c4">

https://github.com/pytorch/pytorch/blob/9a492fc27f3f1f0f71ed8b24704430a87a99cec9/test/test_custom_ops.py#L1459-L1467